core/src/main/java/org/infinispan/interceptors/distribution/BaseDistributionInterceptor.java

```diff
@@ -291,8 +291,8 @@ protected final Object handleNonTxWriteCommand(InvocationContext ctx, DataWriteC
       // TotalOrderStateTransferInterceptor doesn't set the topology id for PFERs.
       // TODO Shouldn't PFERs be executed in a tx with total order?
       boolean topologyChanged = isSync && currentTopologyId != commandTopologyId && commandTopologyId != -1;
-      log.tracef("Command topology id is %d, current topology id is %d, successful? %s",
-            commandTopologyId, currentTopologyId, command.isSuccessful());
+      log.tracef("Command topology id is %s, current topology id is %d, successful? %s",
```

Comment:
Unfortunately jboss logging 3.2 has introduced a bunch of overloaded methods like debugf(String format, int param1, Object param2, Object param3) which are ambiguous with debugf(String format, Object... params);
I'm merely disambiguating, but if you have a better idea, I'm open :)